scrapers/Baberotica.yml

```diff
@@ -2,42 +2,53 @@ name: Baberotica
 sceneByURL:
   - action: scrapeXPath
     url:
-      - avidolz.com
       - baberotica.com
       - baberoticavr.com
-      - teenthais.com
+      - avjiali.com
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
+    common:
+      $performer: //div[@class="cat"]//a[@rel="tag"][contains(@href, ".com/model/")]
     scene:
       Title:
-        selector: //h1[@itemprop="name"]
+        selector: //div[@class="videoinfo"]/h1/text()
       Performers:
-        Name: //p/span[@itemprop="actor"]/a/span
+        Name: $performer/text()
+        URL: $performer/@href
       Date:
-        selector: //div[contains(@class,"pure-u-1")or @class="container"]/meta[@itemprop="datePublished"]/@content
+        selector: //div[@class="video-date"]/text()
         postProcess:
           - replace:
-              - regex: "T.+$"
-                with: ""
-          - parseDate: 2006-01-02
+              - regex: "(\\d)(st|th|nd|rd)"
+                with: $1
+          - parseDate: January 2, 2006
       Tags:
-        Name: //a[@itemprop="genre"]
+        Name: //div[@class="cat"]//a[@rel="tag"][not(contains(@href, ".com/model/"))]/text()
       Details:
-        selector: //div[@itemprop="description"]
+        selector: //div[@class="videoinfo"]/p
+        postProcess:
+          - replace:
+              - regex: '\.\.\. Read More'
+                with: ''
       Image:
-        selector: //div[contains(@class,"pure-u-1")or @class="container"]/meta[@itemprop="thumbnailUrl"]/@content
+        selector: //video[contains(@id,"videohtml5tour")]/@poster
         postProcess:
           - replace:
               - regex: ^
                 with: "https:"
+      Code:
```

Comment:
This works for avjiali but not for baberotica

Reply:
I'm not able to get baberotica codes in the same way. AV Jiali's studio codes happen to be in the URL of the cover image, so I grab them from there, but baberotica doesn't do this. I don't see anywhere else on the page to get it, so I figured it's better to at least have it for avjiali than none at all. All the scenes in stashdb that have codes (from my quick glance) seem to be from the sexlikereal.com domain rather than baberotica.com or baberoticavr.com, so that might have something to do with it.